lib/exe/manifest.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+if (!defined('DOKU_INC')) {
+    define('DOKU_INC', __DIR__ . '/../../');
+}
+require_once(DOKU_INC . 'inc/init.php');
+
```

Comment:
Shouldn't this check if the action is enabled before sending the manifest?

Reply:
Good catch! I'll look into it. If it isn't enabled we send a 400?

Reply:
Should be a 404 I'd say.

Reply:
Fixed in ac6ceee39397df8191bff60624aad343e50fdfe3